packages/flame/lib/src/experimental/layout_component.dart

```diff
@@ -175,6 +175,15 @@ abstract class LayoutComponent extends PositionComponent {
 
   @override
   void onChildrenChanged(Component child, ChildrenChangeType type) {
+    if (child is! PositionComponent) {
```

Comment:
This could be `ReadOnlySizeProvider` instead

Reply:
On the other hand, we're not doing that for the initial setup either, so I guess this will be enough for now. 

Another thing I thought about is that we probably should also listen to the position of the children, since that could move children outside of the shrinkwrap.

Reply:
@spydon but the LayoutComponents are supposed to fully control the positions of its children, no? Like, they align the children in rows or columns, and align them in the center or whatnot. So for any of the children to be positioned in an unexpected way could be considered a misuse of the LayoutComponent.

Reply:
Ah, that's true! Then it is all correct :)